app/controllers/posts.js

```diff
@@ -74,10 +74,10 @@ export default Controller.extend({
     postListFocused: equal('keyboardFocus', 'postList'),
     postContentFocused: equal('keyboardFocus', 'postContent'),
 
-    sortedPosts: computed('model.@each.status', 'model.@each.publishedAt', 'model.@each.isNew', 'model.@each.updatedAt', function () {
```

Comment:
The `model.` bit is still needed here, if you've seen the straight `@each` example somewhere it's likely from when controllers still proxied to the `model` property but Ember hasn't done that for a long time. However, you can use the brace expansion syntax to make it more concise:

```javascript
sortedPosts: computed('model.@each.{isNew,publishedAt,status,updatedAt}', function () {
```